pages/national_resources/de_national_resources.md

```diff
@@ -3,8 +3,8 @@
 title: Germany
 search_exclude: true
```

Comment:
```suggestion
```
this can be deleted if you want to display it on the menu, now that there is content on the page.